run-on-lido/stvaults/node-operators-identification.md

```diff
@@ -0,0 +1,266 @@
+    
+5. **Node Operator card creation**
+    
+    The stVaults Committee creates your Node Operator card (name/logo, links, geo/jurisdiction, infra/DVT/relays, certifications/audits/insurance/bug-bounty). Your card will be displayed on the Node Operators Overview page when you have at least one stVault with at least 32 ETH of Total Value. *The Node Operators Overview page will be released in the weeks following the V3 Mainnet launch.*
+    
+6. **Communication & listing**
+    
+    Your card and tiers grid are set, and you can now continue as an Identified Node Operator. Additionally, every Identified Node Operator will have a direct communication channel with the stVaults Committee representatives in a shared Telegram group. Questions and discussions can also always be raised on the Lido Research Forum.
+    
+
+---
+
+## **How tiers map to risk profile & business case**
+
+**1) Why categories exist.**
+
+The V3 risk assessment framework must balance growth with protection from major risks like correlated slashing and centralization. **Categories** connect qualitative (ops/decentralization) and quantitative (stake volumes) signals so that **stETH minting limits** and **Reserve Ratios** scale with an operator’s overall profile. Defaults keep the protocol safe; categories allow well-earned flexibility.
+
+**Category** is the Node Operators classification; each NO category has an associated default tier grid, which is used by Node Operators from this Category. Each Node Operator has its own Tiers with its own stETH minting limits. 
+
+**2) Why multiple tiers per category?**
+
+Each tier is designed for a distinct business case — not a linear “risk ladder.”
+
+- **First tiers (Tier-1..2):** **lowest RR** + **lowest stETH minting limit** → **liquidity-first** scenarios (strategy builders, looping) where liquidity matters most; stETH minting capacity **is tightly bounded**.
+- **Upper tiers (Tier-3…N):** **more conservative RR** + **much higher stETH minting limit** → **volume-first** scenarios (institutions, funds, ETF/ETP) where large mints — sometimes **on demand** — matter more than high liquidity.
+
+**3) How RR & stETH minting limit interact.**
+
+In any stVault, stETH capacity depends on the Reserve Ratio and the Total Value, but is also constrained by limits:
+
+- stVault stETH limit (equal to Tier stETH limit by default).
+- Tier stETH limit.
+- Node Operator stETH limit.
+- stVaults’ global stETH limit.
+
+![Tiers and minting parameters](/img/stvaults/tiers-and-minting-parameters.png)
+
+## **Governance & operational safety net.**
+
+In order to keep the stVaults risk framework flexible and to streamline the governance process around it, it is proposed that [the stVaults Committee](https://research.lido.fi/t/stvaults-committee-proposal/10608) will be responsible for the following actions:
```

Comment:
Manage core redemptions, trigger validator exits for rebalancing and redemptions, and put vaults in jail?